spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/MM.scala

```diff
@@ -53,6 +53,8 @@ class MM[T: ClassTag](
   override def updateOutput(input: Table): Tensor[T] = {
     var (ma, mb) = checkInputFormat(input)
 
+    output.set
```

Comment:
set will change memory. should we avoid that? Another fix is using ```baddbmm``` instead of ```bmm```. We can set the ```beta == 0``` and ```alpha == 1```.